### DISCUSSION TITLE
Opening Source in Browse Mode

### DISCUSSION THREAD

**Discussion by JonFP** [codefori/vscode-ibmi#594] [IBM i]
Anyone know how to do this?  The Preview option doesn't work since I need to have two or three source members open at the same time for browsing and with preview active it just closes the first one when I open the second.

**Comment by p-behr** [comment]
@JonFP You have a couple of choices: 
When the source member opens in preview mode you can switch out of preview mode with "Ctrl+K Enter" (that's Ctrl+K at the same time, followed by pressing Enter).   
Another option, if you don't generally care about preview mode, is to just turn it off in your settings.  
You get to your settings by pressing F1 and then search for "settings" in the command palette and choosing the "Open Settings (UI)".  You can search the settings for "preview".  You want to uncheck the Workbench > Editor: Enable Preview option.  That will cause each source member to open in it's own tab.

**Reply by JonFP** [reply]
Yes I know I can do that - but that isn't browse mode.  It just switches to edit mode from preview. I can do the same thing by simply typing a space on any line.  But I want the files open in **browse only** mode like I can do in PDM or RDi.
